src/NuGet.Clients/NuGet.PackageManagement.UI/Models/IconBitmapStatus.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.PackageManagement.UI
+{
+    public enum IconBitmapStatus
+    {
+        None = 0,
+        NeedToFetch,
+        Fetching,
+        MemoryCachedIcon,
+        DownloadedIcon,
+        EmbeddedIcon,
+        DefaultIcon,
+        DefaultIconDueToDecodingError,
+        DefaultIconDueToNullStream,
+        DefaultIconDueToNoPackageReader,
+        DefaultIconDueToNetworkFailures,
+        DefaultIconDueToWebExceptionBadNetwork,
+        DefaultIconDueToWebExceptionOther,
+        DefaultIconDueToRelativeUri
+    }
+
+    public static class IconBitmapStatusUtility
```

Comment:
was leaving there because it will need change if the enum gets added to.. but i spit it out, made it internal, and left a comment with the enum. fixed in next iteration.